http3/server.go

```diff
@@ -255,6 +255,9 @@ func (s *Server) serveImpl(startListener func() (quic.EarlyListener, error)) err
 	for {
 		conn, err := ln.Accept(context.Background())
 		if err != nil {
+			if errors.Is(err, quic.ErrServerClosed) {
+				return http.ErrServerClosed
```

Comment:
Why do you convert to the `http.ErrServerClosed` here? The HTTP server might not be closed at this point.

Reply:
Yes, that's right. I reverted it, and there is no particular need to convert in order to reach the goal of this PR.